test/ex_machina/ecto_strategy_test.exs

```diff
@@ -53,4 +53,10 @@ defmodule ExMachina.EctoStrategyTest do
       TestFactory.insert(:user, name: "Maximus") |> TestFactory.insert
     end
   end
+
+  test "create/1 raises helpful error message directing use of insert/1" do
```

Comment:
Ok I can move it, I had only put it there b/c all the `insert/1` tests were in `ecto_strategy_test`. While the `insert/2, insert_pair/2, insert_list/3` tests were in `ex_machina_test`.